src/which.js

```diff
@@ -2,6 +2,9 @@ var common = require('./common');
 var fs = require('fs');
 var path = require('path');
 
+// XP's system default value for PATHEXT system variable
```

Comment:
You can delete environmental variables but if you delete this one I'm guessing shelljs would be the least of your problems :D